pkgs/applications/terminal-emulators/alacritty/default.nix

```diff
@@ -112,6 +107,9 @@ rustPlatform.buildRustPackage rec {
       patchelf --set-rpath "${lib.makeLibraryPath rpathLibs}" $out/bin/alacritty
     ''
   ) + ''
+    for prog in $out/bin/*; do
+      wrapProgram $prog --suffix PATH : "${lib.makeBinPath [ xdg-utils ]}"
+    done
```

Comment:
I think it might be better to be explicit and just add this for `$out/bin/alacritty`